auth/handlers/handle_create-password.go

```diff
@@ -0,0 +1,54 @@
+package handlers
+
+import (
+	"github.com/cortezaproject/corteza-server/auth/request"
+	"github.com/cortezaproject/corteza-server/pkg/auth"
+	"github.com/cortezaproject/corteza-server/system/service"
+	"go.uber.org/zap"
+)
+
+func (h *AuthHandlers) createPasswordForm(req *request.AuthReq) error {
```

Comment:
We could/should/must(?) protect both these endpoints a bit.
Can user that already has password set use this? 
There is `PasswordSet()` fn on auth service that checks this.